tutorials/realm/Barrier.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: page
+permalink: /tutorial/barrier.html
```

Comment:
Can we permalink all of these under  `/tutorial/realm/...`?

Reply:
fixed